### ISSUE TITLE
.env file not loaded automatically

### ISSUE THREAD

**Issue by teristam** [imartinez/privateGPT#57]
With the latest update, when running ` python .\ingest.py` , it reported

```
Traceback (most recent call last):
  File "F:\code\privateGPT\ingest.py", line 34, in <module>
    main()
  File "F:\code\privateGPT\ingest.py", line 27, in main
    llama = LlamaCppEmbeddings(model_path=llama_embeddings_model, n_ctx=model_n_ctx)
  File "pydantic\main.py", line 339, in pydantic.main.BaseModel.__init__
  File "pydantic\main.py", line 1102, in pydantic.main.validate_model
  File "C:\Users\teristam\Anaconda3\envs\private_gpt\lib\site-packages\langchain\embeddings\llamacpp.py", line 64, in validate_environment
    model_path = values["model_path"]
KeyError: 'model_path'
```
I notice the `.env` file is not loaded anywhere in the Python code?

**Comment by sorin** [comment]
PR Here https://github.com/imartinez/privateGPT/pull/58

**Comment by imartinez** [comment]
Fixed in #58. Thanks for reporting.
